server/metrics/collectors/collectors.go

```diff
@@ -0,0 +1,117 @@
+package collectors
+
+import "strings"
+
+// Collector specifies a single metrics collector identifier.
+type Collector string
+
+// Collectors specifies a list of metrics collectors.
+type Collectors []Collector
+
+const (
+	crioPrefix      = "crio_"
+	subsystemPrefix = Subsystem + "_"
```

Comment:
nit: can we define this below Subsystem?

Reply:
Yep, I'll have to rebase in any case if https://github.com/cri-o/cri-o/pull/5059 merges.

Reply:
Fixed